gordo/machine/dataset/datasets.py

```diff
@@ -179,6 +184,7 @@ def __init__(
         self.high_threshold = high_threshold
         self.interpolation_method = interpolation_method
         self.interpolation_limit = interpolation_limit
+        self.filter_periods = filter_periods
```

Comment:
Is better to create actual instance of `FilterPeriods` class here:
```suggestion
        self.filter_periods = FilterPeriods(granularity=self.resolution, **filter_periods) if filter_periods else None
```